tests/integration/plugins/test_jlink.py

```diff
@@ -23,6 +23,71 @@
 from craft_parts import LifecycleManager, Step, errors
 
 
+def test_jlink_plugin_embedded_jar(new_dir, partitions):
+    parts_yaml = textwrap.dedent(
+        """
+        parts:
+            my-part:
+                plugin: jlink
+                source: https://github.com/canonical/chisel-releases
+                source-type: git
+                source-branch: ubuntu-24.04
+                jlink-jars: ["test.jar"]
+                after: ["stage-jar"]
+            stage-jar:
+                plugin: dump
+                source: .
+        """
+    )
+    parts = yaml.safe_load(parts_yaml)
+
+    # build test jar
```

Comment:
can you put this part of the code in a separate function? It's a bit distracting from the actual test

Reply:
Refactored to a fixture, thank you :)